tests/src/test/scala/whisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -365,7 +430,7 @@ class ContainerPoolObjectTests extends FlatSpec with Matchers with MockFactory {
             'second -> freeWorker(second),
             'oldest -> freeWorker(oldest))
 
-        ContainerPool.remove(EntityName(commonNamespace), pool) shouldBe Some('oldest)
+        ContainerPool.remove(first.action, EntityName("anyNamespace"), pool) shouldBe Some('oldest)
```

Comment:
Should use `createAction()`. The relationship to the first warmed data is not necessary here.